docs/src/main/sphinx/release/release-472.md

```diff
@@ -0,0 +1,96 @@
+# Release 472 (26 Feb 2025)
+
+## General
+
+* Color console output for improved logs readability. ({issue}`25090`)
+* Suggest configuration for unused properties in log output. ({issue}`25090`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## DuckDB connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Change default for  `hive.partition-projection-enabled` to `true`. ({issue}`25079`)
```

Comment:
User facing impact @pajaks or @ebyhr ? Does that improve performance? Also.. I think this is not a breaking change .. right?